ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/jsonrpc/response/JsonRpcSuccessResponse.java

```diff
@@ -24,8 +25,16 @@
   private final Object id;
   private final Object result;
 
+  @JsonCreator
```

Comment:
Why do we need this additional constructor?

Reply:
We now _receive_ these from Besu (for Nonce) - thus it needs a constuctor which can be populated from a Json Decoder.